test/commands/core/core.plugins.test.ts

```
@@ -40,6 +40,26 @@ describe('core plugin commands', () => {
         expect(cleanAddStderr).to.contain('Successfully added')
     })
 
+    it('adds a plugin to a collection without --collection by auto-detecting', async function() {
+        this.timeout(30000)
```

ESLint

[error] Invalid group length in numeric value.

(unicorn/numeric-separators-style)